docs/command-reference/commands-details/workload_get.md

```diff
@@ -0,0 +1,244 @@
+# tanzu apps workload get
+
+tanzu apps workload get is a command used to retrieve information and status about a workload.
+
+You can view workload details at anytime in the process. Some details, such as the status of the workload, the source of the workload application, the supply chain which took care of the workload, the supply chain resources which interact with the workload, if there is/was any issue while deploying the workload and finally which *Pods* the workload generates and the knative services related to the workload, if the supply chain is using knative.
+
+## Default view
+
+In the output of workload get command, there is the name of the workload and its status. After this, its source info. If the workload was taken by a supply chain, the info of its name and the status of the workload regarding the supply chain is displayed. Next to this, the existing steps where the workload has been through in the supply chain. In the next section, if there is any issue with the workload, the name and a message explaining what it means is displayed. In the next three sections, the services claims the workload has, its related pods and last its knative services are shown.
```

Comment:
There are multiple sections in workload get command output. Following data is displayed
- Name of the workload and its status. 
- Display source information of workload.
- If the workload was matched with a supply chain, the information of its name and the status is displayed.
- Information and status of the individual steps that's defined in the supply chain for workload.
- Any issue with the workload, the name and  corresponding message.
-  Workload related resource information and status like services claims, related pods, knative services.